### PR TITLE
io_error: cleanup TODO

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Long Dai <long0dai@foxmail.com>

Commit Message:

Rename with a more meaningful name as TODO comments.

Additional Description: Low
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: envoy/api/io_error.h [hunk 1]
```diff
@@ -50,20 +50,20 @@ using IoErrorPtr = std::unique_ptr<IoError, IoErrorDeleterType>;
...
  * according to different implementations.
- * If the call succeeds, ok() should return true and |rc_| is valid. Otherwise |err_|
+ * If the call succeeds, ok() should return true and |return_value_| is valid. Otherwise |err_|
  * can be passed into IoError::getErrorCode() to extract the error. In this
- * case, |rc_| is invalid.
+ * case, |return_value_| is invalid.
  */
 template <typename ReturnValue> struct IoCallResult {
-  IoCallResult(ReturnValue rc, IoErrorPtr err) : rc_(rc), err_(std::move(err)) {}
+  IoCallResult(ReturnValue rc, IoErrorPtr err) : return_value_(rc), err_(std::move(err)) {}
```
Comment (alyssawilk) [inline envoy/api/io_error.h:58]: rc -> return_value ?

**Review by alyssawilk — COMMENTED** [review]
Thanks for the clean up!  Just one minor nit and CI checks and you're good to go!

File: envoy/api/io_error.h (hunk 1 above)
Comment (daixiang0) [reply envoy/api/io_error.h]: Sorry for mistake, fixed.

**Comment by daixiang0** [comment]
sorry for force push after rebase main branch.

**Comment by alyssawilk** [comment]
Just to be aware, we've been making some improvements to Envoy tooling for more prompt reviews, but draft PRs are not included in the tooling.  I'm updating the contribution guidelines to make that clear (https://github.com/envoyproxy/envoy/pull/17063/files) , but as this PR predates the repokitteh warning I just want to call out that draft PRs are likely not going to get fast turnaround. Please consider marking this as ready for review if it stalls, and you want the assignee to take a look!

**Comment by daixiang0** [comment]
@alyssawilk thanks for great work, I would to make it ready for review soon.

**Review by alyssawilk — APPROVED** [review]
Thanks for the clean up!

**Comment by alyssawilk** [comment]
Happy to merge once this is marked as not draft :-)
/wait

**Comment by daixiang0** [comment]
@alyssawilk done.

**Pull request merged**